### DISCUSSION TITLE
Extending share plugin

### DISCUSSION THREAD

**Discussion by seanaye** [ionic-team/capacitor#3693] [Q&A]
I need to extend the share plugin so that I can share rich media to other platform as outlined in this article https://developers.facebook.com/docs/instagram/sharing-to-stories

I have no idea how I would go about doing this although it should be possible. Can someone point me in the right direction? I can't find any docs on doing this with capacitor.

I don't have any experience with native development so even just a tip of what files to create and where to create them would be helpful

**Comment by jcesarmobile** [comment]
Capacitor plugins can’t be extended, you have to create your own plugin for that.
